deploy/crds/k8s.nginx.org_nginxingresscontrollers_crd.yaml

```diff
@@ -57,7 +68,7 @@ spec:
                 type: boolean
               enableSnippets:
                 description: Enable custom NGINX configuration snippets in VirtualServer
-                  and VirtualServerRoute resources. Requires enableCRDs set to true.
+                  and VirtualServerRoute resources.
```

Comment:
Was this change intentional?

Looks like it was omitted in https://github.com/nginxinc/nginx-ingress-operator/blob/d35912cadc0ad181820ff9a60579eba1155c0d35/pkg/apis/k8s/v1alpha1/nginxingresscontroller_types.go#L37-L40 which may lead to updating CRDs via autogeneration removing this.

But looks like you resolved that error in: https://github.com/nginxinc/nginx-ingress-operator/pull/13/files#diff-319625fe824aa0f8d5169f59f7485660

So I think this can be reverted?

Reply:
There was an inconsistency between the code and the CRD. I need to re-generate CRDs I think. There was an inconsistency between the code and the CRD I believe